Resources/Prototypes/_White/Entities/Clothing/Head/modsuit.yml

```diff
@@ -0,0 +1,85 @@
+﻿- type: entity
+  parent: BaseClothingModsuitHelmet
+  id: ClothingModsuitHelmetInfiltrator
+  name: infiltrator modsuit helmet
+  description: A special modular suit helmet designed for compact folding inside modular suit control.
```

Comment:
"A purpose-built helmet that can be neatly folded and store in the infiltrator suit's main unit."